src/app/issue.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
```

Comment:
Could you match the import styles up please

Reply:
There are a mix of styles throughout the files

Reply:
Fixed by d03d1873c0aba72e6776df6be3258249f86e5153